### ISSUE TITLE
country badge idea

### ISSUE THREAD

**Issue by dalekunce** [AmericanRedCross/osm-stats-workers#40]
- level 1 badge for each country mapped. 
- level 2 when you have 100 edits
- level 3 when you have 1000 edits

**Comment by danbjoseph** [comment]
so if i've mapped in 73 countries i have 73 badges (of various levels)?

**Comment by dalekunce** [comment]
@danbjoseph yep, this wouldn't be to dissimilar to how pascal does the country listing on his website. With the idea of encouraging local edits.

**Comment by danbjoseph** [comment]
to encourage local edits, i'd think you'd want something like having people somehow setting their "home" country and then rewarding them for edits in that place.

**Comment by danbjoseph** [comment]
this would give me 73 badges and the USA isn't at the top of my edits list
